docker-compose.es-cluster.yml

```diff
@@ -0,0 +1,95 @@
+version: '3.7'
+
+x-logging:
+  &default-logging
+  driver: "json-file"
+  options:
+    max-size: "10m"
+    max-file: "3"
+
+services:
+  elasticsearch:
+    image: semtech/mu-search-elastic-backend:1.1.0
+    volumes:
+      - ./data/elasticsearch/:/usr/share/elasticsearch/data
+      - ./data/elastic_cluster_shared_folder:/shared_folder
+    environment:
+      - "node.name=elasticsearch"
+      - "bootstrap.memory_lock=false"
+      - "cluster.name=es-cluster"
+      - "discovery.seed_hosts=elasticsearch2,elasticsearch3"
+      - "cluster.initial_master_nodes=elasticsearch,elasticsearch2,elasticsearch3"
+      - "ES_JAVA_OPTS=-Xms512m -Xmx512m -Des.index.number_of_replicas=0 -Des.enforce.bootstrap.checks=true"
+      - "xpack.security.enabled=false"
+      - "xpack.security.http.ssl.enabled=false"
+      - "xpack.security.transport.ssl.enabled=false"
+      - "xpack.ml.enabled=false"
+      - "xpack.graph.enabled=false"
+      - "xpack.watcher.enabled=false"
+      - "path.repo=/shared_folder"
+      - "discovery.type="
```

Comment:
this is the only way to disable single-node inherited from the parent docker-compose file